code-challenges.js

```diff
@@ -39,26 +73,41 @@ var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 
 var vowelTester1 = "learn"
 // Expected output: 1
-var vowelTester1 = "throw"
+var vowelTester2 = "throw"
 // Expected output: 3
 
+const firstSrtringVowel = (str) =>{
 
-
-
-
+    var stringArray = str.split('') //[ 't', 'h', 'r', 'o', 'w' ]
+    var firstVowel = stringArray.findIndex(vowel => vowel === 'a' || vowel === 'e' ||vowel === 'i' || vowel === 'o'|| vowel === 'u')
+    return firstVowel
+}
```

Comment:
interesting use of `.findIndex` ! you can also just return the second line if you'd like
ex: `return stringArray.findIndex(vowel => vowel === 'a' ...`